src/twisted/trial/test/test_matchers.py

```diff
@@ -85,3 +87,39 @@ def test_ioerror(self) -> None:
                 contains_string(repr(p.path)),
             ),
         )
+
+
+class EmptyInterface(Interface):
+    ...
+
+
+class NonemptyInterface(Interface):
+    def required_method() -> None:
+        ...
+
+
+class ProvidesTests(SynchronousTestCase):
+    """
+    Tests for L{fileContents}.
+    """
+
+    def test_matches_an_implementer_instance(self) -> None:
+        @implementer(EmptyInterface)
+        class EmptyClass:
+            ...
+
+        assert_that(provides(EmptyInterface).matches(EmptyClass()), equal_to(True))
```

Comment:
I am not familiar with matchers.
But I fell the final `equals_to(True)` or `equals_to(False` is verbose

I was expecting something like
```diff
- assert_that(provides(EmptyInterface).matches(EmptyClass()), equal_to(True))
+ assert_that(providers(EmptyInterface).matches(EmptyClass()))
+ assert_that(not().providers(EmptyInterface).matches(EmptyClass()))
```